lib/private/Files/Storage/Wrapper/Encryption.php

```diff
@@ -511,7 +451,7 @@ protected function verifyUnencryptedSize(string $path, int $unencryptedSize): in
 	 *
 	 * @return int calculated unencrypted size
 	 */
-	protected function fixUnencryptedSize(string $path, int $size, int $unencryptedSize): int {
+	protected function fixUnencryptedSize(string $path, int $size, int $unencryptedSize): float|int {
```

Comment:
```suggestion
	protected function fixUnencryptedSize(string $path, int $size, int $unencryptedSize): int|float {
```
Please always use the order int|float to help with `grep` searches.